pkg/interop/runtime/runtime.go

```diff
@@ -96,3 +96,9 @@ func GetInvocationCounter() int {
 func Platform() []byte {
 	return neogointernal.Syscall0("System.Runtime.Platform").([]byte)
 }
+
+// GetRandom returns pseudo-random number which depends on block nonce and tx hash.
```

Comment:
I think we should somehow tell that this number is different each time the function is called.